src/utils/axios.ts

```diff
@@ -0,0 +1,23 @@
+import axios, {
+  AxiosInstance,
+  AxiosRequestConfig,
+  AxiosError,
+} from 'axios';
+
+const baseConfig: AxiosRequestConfig = {
+  timeout: 3000,
+  withCredentials: true,
+};
+
+type GetAxiosInstance = (baseURL?: string) => AxiosInstance;
+
+const getAxiosInstance: GetAxiosInstance = (baseURL = '/') => (
+  axios.create({
+    ...baseConfig,
+    baseURL,
+  })
+);
+
+const { isAxiosError } = axios;
+export default getAxiosInstance;
+export { AxiosError, isAxiosError };
```

Comment:
```suggestion
export const { isAxiosError } = axios;
export default getAxiosInstance;
export { AxiosError };
```
А вообще смысла такого реэкспорта немного, все по ссылке передается.
А в AuthApi так и так идет импорт из пакета axios